src/main.py

```diff
@@ -0,0 +1,45 @@
+import os
+from dotenv import load_dotenv
+from langchain.agents import load_tools
+from langchain.agents import initialize_agent
+from langchain.llms import OpenAI
+
+from langchain.memory import ConversationBufferMemory
+
+# .envからAPI_KEYの読み込み
+load_dotenv()
+temp = os.getenv("temperature")
+
+
+# llmとagent定義
+
+
+llm = OpenAI(temperature=temp)
+tools = load_tools(["llm-math", "python_repl"], llm=llm)
+agent = initialize_agent(tools, llm, AgentType="zero-shot-react-description")
+
+
+memory = ConversationBufferMemory(
+    human_prefix="User", ai_prefix="Bot", memory_key="history", return_messages=True
+)
+
+
+# 会話を開始
+
+while True:
```

Comment:
```suggestion
while True:
```
`33`行目でbreakする以外で、whileを抜ける方法がないように見えるのは意図通りでしょうか？
ここからまだ編集する予定であれば問題ありません
もし編集がある場合は、よければ、`todo`をコメントアウトでつけておくとReviewするときに、気にしなくて済むやもしれません！